scripts/reset-password.js

```diff
@@ -81,6 +81,7 @@ mg.connect(function (db) {
       if (processedCount === 0) {
         console.log(chalk.yellow('No users were found.'));
       } else {
+        /* eslint  no-nested-ternary:0 */
```

Comment:
That was the easiest way, not to interfere with the code too much. If you can provide `if`'`s version of this, I'll replace it.